producers-commissioning/src/main/java/uk/gov/moj/sdt/producers/comx/config/bulkfeedback/BulkFeedbackConfigBOne.java

```diff
@@ -0,0 +1,171 @@
+                                                                                     "mcolClaimStatusUpdate",
+                                                                                     "Initially Accepted",
+                                                                                     "DUP_CUST_REQID", "", "", ""),
+                                                                                     Lists.newArrayList("USER_REQUEST_ID_B5",
+                                                                                     "mcolJudgment",
+                                                                                     "Rejected",
+                                                                                     "24",
+                                                                                     "This judgment request is invalid on the referenced claim.", ""),
+                                                                                     Lists.newArrayList("USER_REQUEST_ID_B6",
+                                                                                     "mcolWarrant",
+                                                                                     "Initially Accepted",
+                                                                                     "", "",
+                                                                                     """
+                                                                                                            <![CDATA[
+                                                                                                                                      <ind:mcolResponseDetail xmlns:ind="http://ws.sdt.moj.gov.uk/2013/sdt/BulkFeedbackResponseSchema" xmlns:mresp="http://ws.sdt.moj.gov.uk/2013/mcol/ResponseDetailSchema">
+                                                                                                                                         <mresp:warrantNumber>0Z000150</mresp:warrantNumber>
+                                                                                                                                         <mresp:enforcingCourtCode>127</mresp:enforcingCourtCode>
+                                                                                                                                         <mresp:enforcingCourtName>BIRMINGHAM</mresp:enforcingCourtName>
+                                                                                                                                         <mresp:fee>10000</mresp:fee>
+                                                                                                                                      </ind:mcolResponseDetail>
+                                                                                                                                    ]]>
```

Comment:
Lots of extra indentation inside the text block.  Reducing this would make it more readable.

Reply:
done this iindentation